superset-frontend/cypress-base/cypress/integration/explore/visualizations/treemap.test.js

```diff
@@ -1,96 +0,0 @@
-/**
- * Licensed to the Apache Software Foundation (ASF) under one
- * or more contributor license agreements.  See the NOTICE file
- * distributed with this work for additional information
- * regarding copyright ownership.  The ASF licenses this file
- * to you under the Apache License, Version 2.0 (the
- * "License"); you may not use this file except in compliance
- * with the License.  You may obtain a copy of the License at
- *
- *   http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing,
- * software distributed under the License is distributed on an
- * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
- * KIND, either express or implied.  See the License for the
- * specific language governing permissions and limitations
- * under the License.
- */
-describe('Visualization > Treemap', () => {
```

Comment:
I removed the test because it's not applicable anymore. By default, charts are rendered using canvas. Even if we created a test configuration to render the charts using SVG, the groupby colors are not static anymore, they suffer small variations depending on the value. This type of test is more suitable with visual regression testing using Applitools or Chromatic, which is not in the scope of this PR.